### PR TITLE
Expand console support for axaml

### PR DESCRIPTION
**Pull request by crobibero**
### Description:

Fixes https://github.com/Xavalon/XamlStyler/issues/445

### Checklist:
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes
* [x] I have tested my changes by running the extension in VS2017
* [x] I have tested my changes by running the extension in VS2019
* [x] I have tested my changes by running the extension in VS2022
* [x] If changes to the [documentation](https://github.com/Xavalon/XamlStyler/wiki) are needed, I have noted this in the description above

### CODE REVIEW

**Review by grochocki — APPROVED** [review]

File: src/XamlStyler.Console/XamlStylerConsole.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@ namespace Xavalon.XamlStyler.Console
 {
     public sealed class XamlStylerConsole
     {
+        private static readonly string[] SupportedPatterns = { "*.xaml", "*.axaml" };
```
Comment (grochocki) [inline src/XamlStyler.Console/XamlStylerConsole.cs:15]: The asterisk wildcard is required for retrieving all matching XAML and AXAML files in a specified directory (line 192) but cannot be there when checking the file extension when processing of the file itself (line 241).

**Review by grochocki — CHANGES_REQUESTED** [review]
Need to update extension patterns to support both finding files in a directory and then processing each file.

File: src/XamlStyler.Console/XamlStylerConsole.cs (hunk 1 above)
Comment (crobibero) [reply src/XamlStyler.Console/XamlStylerConsole.cs:15]: Sorry, thought I had tested that

**Review by grochocki — APPROVED** [review]
That did the trick! Thanks again :)

**Pull request merged**